README.md

```diff
@@ -85,7 +85,7 @@ Open Government 3.0 licence.
 
 ## Contributing
 
-[To learn how to help us build GOV.UK Frontend, see our contribution guidelines.](https://github.com/alphagov/govuk-frontend/blob/f07569c9e7724606970c0d2764f1e9ce7fa14092/CONTRIBUTING.md)
+[To learn how to help us build GOV.UK Frontend, see our contribution guidelines.](https://github.com/alphagov/govuk-frontend/blob/main/CONTRIBUTING.md)
```

Comment:
This works fine as a solution, but if we wanted to avoid tying it to a specific branch we could also make it [a relative link](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#relative-links):

```suggestion
[To learn how to help us build GOV.UK Frontend, see our contribution guidelines.](CONTRIBUTING.md)
```